src/Native/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ if(WIN32)
     add_compile_options(/FC) # use full pathnames in diagnostics
     add_compile_options(/DEBUG)
     add_compile_options(/GS)
+    add_compile_options(/Gz)
```

Comment:
Should we only do this for x86? Like we do in corefx?

[https://github.com/dotnet/corefx/blob/c1788c695d4349c522fc5620eff7d3dd46a0ab5f/src/Native/Windows/CMakeLists.txt#L47-L49](https://github.com/dotnet/corefx/blob/c1788c695d4349c522fc5620eff7d3dd46a0ab5f/src/Native/Windows/CMakeLists.txt#L47-L49)